Open_Science_New

```diff
@@ -576,18 +560,18 @@
 
 	  <!-- Cuban Brain Mapping -->
 	  <section data-transition="convex">
-	    <a href="" target="_blank"><img src="images/Cuban_brain_mapping.png" width="120%"></a>
+	    <a href="https://journals.sagepub.com/doi/abs/10.1177/155005941104200303?rfr_dat=cr_pub%3Dpubmed&url_ver=Z39.88-2003&rfr_id=ori%3Arid%3Acrossref.org&journalCode=eegb"><img src="images/Cuban_brain_mapping.png" width="120%"></a>
```

Comment:
the URL is : 
https://doi.org/10.1177/155005941104200303
Pro tip: always use a DOI link if you can for almost *anything*

Reply:
thanks! this was removed so N/A